src/main/java/org/jsoup/select/Evaluator.java

```diff
@@ -51,6 +51,28 @@ public String toString() {
         }
     }
 
+
+    /**
+     * Evaluator for tag name that ends with
+     */
+    public static final class TagEndsWith extends Evaluator {
+        private String tagName;
+
+        public TagEndsWith(String tagName) {
+            this.tagName = tagName;
+        }
+
+        @Override
+        public boolean matches(Element root, Element element) {
+            return (element.tagName().endsWith(tagName));
+        }
+
+        @Override
+        public String toString() {
+            return String.format("%s", tagName);
```

Comment:
Following the convention for the Tag Evaluator: https://github.com/er1c/jsoup/blob/5d00201aec621e81a37447e32baf1a82040ee8e4/src/main/java/org/jsoup/select/Evaluator.java#L49  I believe this technically handles "null" more appropriately for display purposes:

```
java> String.format("%s", null)
java.lang.String res0 = "null"
```